jest.config.js

```diff
@@ -0,0 +1,18 @@
+const packageJSON = require('./package.json');
+
+process.env.TZ = 'UTC';
+
+module.exports = {
+  verbose: true,
+  name: packageJSON.name,
+  displayName: packageJSON.name,
+  transform: {
+    '\\.[jt]sx?$': 'babel-jest',
+  },
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
```

Comment:
I've left tsx? there because it helps to my next step of moving everything to TS